modules/ROOT/pages/dw-operators.adoc

```diff
@@ -370,3 +367,359 @@ xref:dataweave-variables.adoc#example_local_variable[Examples: Local DataWeave V
+
+.Input
+[source,json,linenums]
+----
+{
+    "name": "Ken",
+    "lastName":"Shokida",
+    "age": 30,
+    "addresses": [{
+        "street": "Amenabar",
+        "zipCode": "AB1234"
+    }]
+}
+----
+
+.Source
+[source,weave,linenums]
+----
+%dw 2.0
+output application/json
+---
+payload update {
+    case s at .addresses[0] -> {
+        "street": "Second Street",
+        "zipCode": "ZZ123"
+    }
+}
+----
+
+.Output
+[source,json,linenums]
+----
+{
+  "name": "Ken",
+  "lastName": "Shokida",
+  "age": 30,
+  "addresses": [
+    {
+      "street": "Second Street",
+      "zipCode": "ZZ123"
+    }
+  ]
+}
+----
+
+==== Attribute Selector
+
+This matching selector allows to match any attribute value. +
+
+The following example updates an attribute value and changes it to upper case:
+
+.Input
+[source,xml,linenums]
+----
+<user name="leandro"/>
+----
+
+.Source
+[source,weave,linenums]
+----
+%dw 2.0
+output application/json
+---
+payload update {
+    case name at .user.@name -> upper(name)
+}
+----
+
+.Output
+[source,xml,linenums]
+----
+<?xml version='1.0' encoding='UTF-8'?>
+<user name="LEANDRO"/>
+----
+
+==== Dynamic Selector
+
+Any of the given selectors can use an expression to make it dynamic. +
+
+The following example selects and updates a value where the field to be updated is in a variable and not hardcoded. In this case from the variable `theFieldName` with value `name`:
+
+.Input
+[source,json,linenums]
+----
+{
+    "name": "Ken",
+    "lastName":"Shokida"
+}
+----
+
+.Source
+[source,weave,linenums]
+----
+%dw 2.0
+output application/json
+var theFieldName = "name"
+---
+payload update {
+    case s at ."$(theFieldName)" -> "Shoki"
+}
+----
+
+.Output
+[source,json,linenums]
+----
+{
+  "name": "Shoki",
+  "lastName": "Shokida"
+}
+----
+
+=== Conditional Update
+
+The update operator has a syntax to write a conditional update if you only want to modify a field under a given condition. +
+
+The following example updates the category based on the name of the user:
+
+.Input
+[source,json,linenums]
+----
+[{"name": "Ken", "age": 30}, {"name": "Tomo", "age": 70}, {"name": "Kajika", "age": 10}]
+----
+
+.Source
+[source,weave,linenums]
+----
+%dw 2.0
+output application/json
+---
+payload map ((user) ->
+    user update {
+        case name at .name if(name == "Ken") -> name ++ " (Leandro)"
+        case name at .name if(name == "Tomo") -> name ++ " (Christian)"
+    }
+)
+----
+
+
+.Output
+[source,json,linenums]
+----
+[
+  {
+    "name": "Ken (Leandro)",
+    "age": 30
+  },
+  {
+    "name": "Tomo (Christian)",
+    "age": 70
+  },
+  {
+    "name": "Kajika",
+    "age": 10
+  }
+]
+----
+
+=== Upserting
+
+The update operator allows to insert a field if it is not present using the `!` symbol at the end of selector expression. When the field is not present the value `null` is going to be bound to the variable. +
+
+The following example updates the category based on the name of the user:
```

Comment:
So...I don't understand how this sentence relates to upserting, and I don't see the `!` symbol.
I expect the example to illustrate the conceptual info above it.